ee/codegen/src/generators/workflow-value-descriptor.ts

```diff
@@ -107,40 +107,37 @@ export class WorkflowValueDescriptor extends AstNode {
   private convertOperatorType(
     workflowValueDescriptor: WorkflowValueDescriptorType
   ): OperatorMapping {
-    if (this.isExpression(workflowValueDescriptor)) {
-      const operator = workflowValueDescriptor.operator;
-      const operatorMappings: Record<string, OperatorMapping> = {
-        "=": "equals",
-        "!=": "does_not_equal",
-        "<": "less_than",
-        ">": "greater_than",
-        "<=": "less_than_or_equal_to",
-        ">=": "greater_than_or_equal_to",
-        contains: "contains",
-        beginsWith: "begins_with",
-        endsWith: "ends_with",
-        doesNotContain: "does_not_contain",
-        doesNotBeginWith: "does_not_begin_with",
-        doesNotEndWith: "does_not_end_with",
-        null: "is_null",
-        notNull: "is_not_null",
-        in: "in",
-        notIn: "not_in",
-        between: "between",
-        notBetween: "not_between",
-      };
-      const value = operatorMappings[operator];
-      if (!value) {
-        throw new NodePortGenerationError(
-          `This operator: ${operator} is not supported`
-        );
-      }
-      return value;
-    } else {
-      throw new NodeAttributeGenerationError(
-        `Operators should exist on expression and not be null`
-      );
+    if (!this.isExpression(workflowValueDescriptor)) {
+      return "equals"; // default operator if not an expression
```

Comment:
[oos] long term, we'll probably want an identity operator to use as the default. maybe that's the same as the group expression.